integration-tests/spring-pojo-connection-limit-test/src/test/resources/microservice.yaml

```diff
@@ -17,7 +17,7 @@
 
 APPLICATION_ID: pojotest-it
 service_description:
-  name: pojo-connection-limit
+  name: pojo
```

Comment:
conflict with integration-tests/pojo-test

Reply:
at last, we will move integration test to real SC, not a mock SC, your logic will cause conflict in that time.